spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -1063,6 +1063,14 @@ def setup_editor(self,
 
         self.set_strip_mode(strip_mode)
 
+    def update_classfuncdropdown(self, symbols):
```

Comment:
```suggestion
    def _update_classfuncdropdown(self, symbols):
        """Update class/function dropdown."""
```

Also, please move this method to be below the `process_symbols` one.